nttt/tidyup.py

```diff
@@ -43,7 +43,7 @@ def increase_indent(self, flow=False, indentless=False):
                                width=1000000)
 
 
-def fix_step(src, lang, dst, disable=()):
+def fix_step(src, lang, dst, disable=(), logging="off"):
```

Comment:
I removed it, but tests fail. It is needed to be transferred through here to trim_md_tags/trim_tags methods so that it is applied. So I added it again